modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,441 @@
+----
+
+Create a cluster with a specific AWS region.
+
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=mycluster --region=us-east-2
+----
+
+[id="rosa-create-idp_{context}"]
+== create idp
+
+Adds an identity provider (IDP) to define how users will log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create idp --cluster=<clusterName> [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to which the IDP will be added.
+
+|--ca
+|The path (string) to PEM-encoded certificate file to use when making requests to the server.
+
+|--client-id
+|The client ID (string) from the registered application.
+
+|--client-secret
+|The client secret (string) from the registered application.
+
+|--mapping-method
+|Specifies how new identities (string) are mapped to users when they log in. Default: claim
+
+|--name
+|The name (string) for the identity provider.
+
+|-t, --type
+|The type (string) of identity provider. Options: github, gitlab, google, ldap, openid
+|===
+
+.Github arguments
+|===
+|Option |Definition
+
+|--hostname
+|(Github) The optional domain (string) to use with a hosted instance of GitHub Enterprise.
+
+|--host-url
+|(Github) The host URL (string) of a GitLab provider. Default: https://gitlab.com
+
+|--organizations
+|(Github) Specifies the organizations for log in access. Only users that are members of at least one of the listed organizations (string) will be allowed to log in.
+
+|--teams
+|(Github) Specifies the teams for log in access. Only users that are members of at least one of the listed teams (string) will be allowed to log in. The format is <org>/<team>.
+|===
+
+.Google arguments
+|===
+|Option |Definition
+
+|--hosted-domain
+|(Google) Restricts users to a Google Apps domain (string).
+|===
+
+.LDAP arguments
+|===
+|Option |Definition
+
+|--bind-dn
+|(LDAP) The DN (string) to bind with during the search phase.
+
+|--bind-password
+|(LDAP) The password (string) to bind with during the search phase.
+
+|--email-attributes
+|(LDAP) The list (string) of attributes whose values should be used as the email address.
+
+|--id-attributes
+|(LDAP) The list (string) of attributes whose values should be used as the user ID. Default: dn
+
+|--insecure
+|(LDAP) Does not make TLS connections to the server.
+
+|--name-attributes
+|(LDAP) The list (string) of attributes whose values should be used as the display name. Default: cn
+
+|--url
+|(LDAP) An RFC 2255 URL (string) which specifies the LDAP search parameters to use.
+
+|--username-attributes
+|(LDAP) The list (string) of attributes whose values should be used as the preferred username. Default: uid
+|===
+
+.OpenID arguments
+|===
+|Option |Definition
+
+|--email-claims
+|(OpenID) The list (string) of claims to use as the email address.
+
+|--extra-scopes
+|(OpenID) The list (string) of scopes to request, in addition to the 'openid' scope, during the authorization token request.
```

Comment:
Suggest not using single quotes here, same throughout a few other places in descriptions below

Reply:
Done.